scripts/service-startup-check.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+# zenoss-inspector-info
+# zenoss-inspector-tags os verify
+# zenoss-inspector-deps serviced-service-list-v.sh
+
+def main():
+    with open("serviced-service-list-v.sh.stdout", 'r') as f:
+        lines = f.readlines()
+    for line in lines:
+        if "Startup" and "/opt/zenoss/ZenPacks" in line:
+            print 'There is a problem with the Startup line for the following:'
```

Comment:
I think we should try to identify the parent service.
The output from serviced-service-list is a JSON array, so instead of parsing the raw the ASCII, you could marshall the file contents into an array of JSON objects, then it would trivial to identify the services by name and/or service ID.